src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$main-size: 100px;
+$main-gap: 1px;
+$day-bg-color: #eee;
+$day-hover-bg-color: #ffbfcb;
```

Comment:
Color literals like `#eee` and `#ffbfcb` should be assigned to variables with meaningful names that reflect their purpose or usage context, ensuring maintainability and readability of the code.